src/services/graphqlQueries.ts

```diff
@@ -129,6 +129,9 @@ const routePathQueryFields = `
     suupaapaik
     suukuka
     suupaapaikr
+    suupituus
+    suuhis
```

Comment:
What are we using suuhis for?

Reply:
Nothing. I checked and the only values are null or 0.. So not sure what this is for :)
Removing it.